modules/lfs/http_client.go

```diff
@@ -179,8 +179,17 @@ func (c *HTTPClient) performOperation(ctx context.Context, objects []Pointer, dc
 				}
 			}
 		} else {
+			var link *Link
```

Comment:
According to Golang syntax, this line could be removed.

```suggestion
			var link *Link
```